libs/judicial-system/types/src/lib/case.ts

```diff
@@ -459,3 +467,68 @@ export function hasCaseBeenAppealed(theCase: Case): boolean {
       Boolean(theCase.prosecutorPostponedAppealDate))
   )
 }
+
+export function getAppealInfo(theCase: Case): Case {
```

Comment:
Consider moving this logic to the the transformer in the api project as it is only used there.

Reply:
Discussed offline, we'll keep it here for now as I'm using part of the logic in my next PR in the backend, and then move later if needed.